.github/workflows/templates/publish-docs-on-release.yml

```diff
@@ -1,8 +1,6 @@
 name: Deploy Documentation on Release
 
 on:
-  release:
```

Comment:
Doc building process is integrated during release-pre-release. Hence no longer needed.